backend/build/api/v1/mentorShip/controllers/userController.d.ts

```diff
@@ -0,0 +1,3 @@
+import { Request, Response } from 'express';
+export declare const getAll: (req: Request, res: Response) => Promise<Response<any, Record<string, any>>>;
+export declare const getSingle: (req: Request, res: Response) => Promise<Response<any, Record<string, any>>>;
```

Comment:
Lets turn this into a custom interface without the use of any. The incoming type will be the data sent from the onboarding process